roles/edpm_kernel/tasks/kernelargs.yml

```diff
@@ -100,13 +100,23 @@
         - /boot/efi/EFI/fedora
 
     - name: Generate grub config
-      ansible.builtin.command: "grub2-mkconfig -o /boot/grub2/grub.cfg"
+      ansible.builtin.command: >-
```

Comment:
We could have checked something like below to not make it distro specific, but this looks good to me for the time being.

if [[ $(grub2-mkconfig --help) =~ '--update-bls-cmdline' ]] ...